src/Controls/tests/TestCases.Shared.Tests/Tests/DragAndDropUITests.cs

```diff
@@ -61,19 +93,61 @@ public void DragAndDropBetweenLayouts()
 			App.WaitForElement("Red");
 			App.WaitForElement("Green");
 			App.DragAndDrop("Red", "Green");
-			App.WaitForElement("DragEventsLabel");
 
-			var textAfterDrag = App.FindElement("DragEventsLabel").GetText();
-			if (string.IsNullOrEmpty(textAfterDrag))
+			App.WaitForElement("DragStartEventsLabel");
```

Comment:
The tests are failing on macOS.

```
DragAndDropBetweenLayouts
Assert.That(rainbowColorText, Is.EqualTo("RainbowColorsAdd:Red"))
Expected string length 20 but was 18. Strings differ at index 17.
Expected: "RainbowColorsAdd:Red"
But was:  "RainbowColorsAdd: "
```

<img width="233" alt="image" src="https://github.com/user-attachments/assets/68c03e0b-1100-4c95-a37a-b07162f09646">

Reply:
@jfversluis  Windows and Mac test cases are randomly failing on CI machines, even though they work properly in local machines. The programmatic drag-and-drop operation using App.DragAndDrop(source, target) is not functioning correctly on Windows and Mac in the CI environment. It seems that the pointer offset coordinates on these devices differ in the CI. Do you have any suggestions for resolving this kind of issue?